src/main.py

```diff
@@ -3,17 +3,23 @@
 
 
 def get_arguments():
-	parser = argparse.ArgumentParser()
-	parser.add_argument('email')
-	parser.add_argument('password')
-	return parser.parse_args()
+    parser = argparse.ArgumentParser()
+    parser.add_argument('driver_path')
+    parser.add_argument('email')
+    parser.add_argument('password')
+    return parser.parse_args()
 
 
 def main():
-	credentials = get_arguments()
-	yad2 = Yad2()
-	yad2.login(credentials.email, credentials.password)
-	yad2.bounce_all_ads()
+    arguments = get_arguments()
+    yad2 = Yad2(arguments.driver_path)
+    try:
+        yad2.login(arguments.email, arguments.password)
```

Comment:
How about logging out?
I'd write a context manager for the login function
`with yad2.login(arguments.email, arguments.password):
yad2.bounce_all_ads()`